atlp-rwanda/cabal-phantom-frontend
# #2rYGYrJC- setup react project

### What does this PR do?
  It create setup of react project

 ### Description of Task to be completed?
 - It will display message under your IP address as server
 - It can be accessed in every device your are connected on same network(ex: wifi) by copy url provided and past it in Other device (ex: your phone if its on same network with your computer)

 ### How should this be manually tested?
 - clone this repository if you don't have it
 - Open your VS Terminal and type in `git checkout origin/ch-setup-react-project-2rYGYrJC`
 - Also Type in `npm install` for installing dependencies
 - Then type `npm run dev` to start view what codes does in browser
 - Try to change message written in `src/component/app.js` and see what happen in browser
   - **_OR_**
- open this[ Link](https://cabal-fronte-ch-setup-r-al0ziv.herokuapp.com/) to have view
 
###  What are the relevant pivotal tracker stories?
[2rYGYrJC](https://trello.com/c/2rYGYrJC/25-setup-react-project)
